scipy/interpolate/tests/test_bsplines.py

```diff
@@ -206,6 +206,17 @@ def test_default_extrap(self):
         yy = b(xx)
         assert_(not np.all(np.isnan(yy)))
 
+    def test_periodic_extrap(self):
+        t = np.sort(np.random.random(10))
```

Comment:
Please seed the generator for reproducibility: `np.random.seed(1234)` (or any other integer)